content/docs/command-reference/push.md

```diff
@@ -151,21 +151,15 @@ $ dvc push data.zip.dvc
 ## Example: With dependencies
 
 Demonstrating the `--with-deps` option requires a larger example. First, assume
-a [pipeline](/doc/command-reference/pipeline) has been setup with these
+a [pipeline](/doc/command-reference/dag) has been setup with these
 [stages](/doc/command-reference/run):
 
 ```dvc
-$ dvc pipeline show
-data/Posts.xml.zip.dvc
-Posts.xml.dvc
-Posts.tsv.dvc
-Posts-test.tsv.dvc
-matrix-train.p.dvc
-model.p.dvc
-Dvcfile
+test-posts
+matrix-train
 ```
```

Comment:
1. Again, "for now just replace this whole block for just a list of stage names in the previous paragraph" (from https://github.com/iterative/dvc.org/pull/1584#discussion_r458603760) so no need for the `dvc` code block anymore.
2. re `I could only figure out these 2 stage names` - we're missing some... I only indicated those 2 specific ones but you can name the other ones whatever you think is best. The hypothetical example should have at least 1 or 2 more stages before `test-posts` for the later text in https://github.com/iterative/dvc.org/pull/1591/files#diff-6c43097b66e96a5f63ac20713879d96f to make sense.

Reply:
Also, please remove lines 168 and 187 while at it 🙂

Reply:
> The hypothetical example should have at least 1 or 2 more stages before test-posts for the later text in https://github.com/iterative/dvc.org/pull/1591/files#diff-6c43097b66e96a5f63ac20713879d96f to make sense.

Oh, that's right. Forgot about the later `middle of the pipeline` text. I've added some more stage names.

Reply:
There is an issue here as stated in [#1584(comment)](https://github.com/iterative/dvc.org/pull/1584#issuecomment-663359221).